pages/index.tsx

```diff
@@ -59,10 +63,7 @@ const Home: NextPage = () => {
           target="_blank"
           rel="noopener noreferrer"
         >
-          Powered by{' '}
-          <span className={styles.logo}>
-            <Image src="/vercel.svg" alt="Vercel Logo" width={72} height={16} />
-          </span>
+          
         </a>
       </footer>
```

Comment:
I know I said not to before, but can we remove everything in index.ts? That way we dont have things we don't need on our repo